docs/content/Schema/Reference/schema-execution-environment.md

```diff
@@ -121,10 +121,10 @@ cubes and cube members. There are reserved property names inside `cube`
 definition that undergo reference resolve transpiling process:
 
 - `sql`
-- `measureReferences`
-- `dimensionReferences`
-- `segmentReferences`
-- `timeDimensionReference`
+- `measures`
+- `dimensions`
+- `segments`
+- `timeDimension`
 - `drillMembers`
 - `drillMemberReferences`
```

Comment:
@keydunov @paveltiunov What is `drillMemberReferences`? This is the only reference to it in all the documentation